query.go

```diff
@@ -888,7 +888,7 @@ func (s *ChainService) GetCFilter(blockHash chainhash.Hash,
 
 	// We didn't get the filter from the DB, so we'll try to get it from
 	// the network.
-	query, err := s.prepareCFiltersQuery(blockHash, filterType, options...)
```

Comment:
Commit message says `queue` instead of `query`.

Reply:
fixed 🤓